code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -0,0 +1,250 @@
+			return
+
+	existing_targets = list()
+	if(!targets.len)
+		to_chat(user, "<span class='warning'>No target found in range!</span>")
+		return
+	spark_setup()
+	user.say(pick("My hate has made me powerful...","I AM THE SENATE!!", "If you will not be turned... you will be destroyed!!", "You will pay the price for your lack of vision!!", "Young fool, only now, at the end, do you understand.", "Your feeble skills are no match for the power of the Dark Side!", "UNNNNLIMITEDDDD POWAHHHH!!", "And now young Skywalker, you will die."))
+	var/mob/living/carbon/target = targets[1]
+	if(get_dist(user,target)>range)
+		to_chat(user, "<span class='warning'>[target.p_theyre(TRUE)] too far away!</span>")
+		return
+
+	playsound(get_turf(user), 'sound/magic/lightningbolt.ogg', 50, TRUE)
+	user.Beam(target,icon_state="lightning[rand(1,12)]",time=5)
+
+	var/damage = rand(min_damage,max_damage)
+
+	if(isliving(user))
+		var/mob/living/L = user
+		L.adjustStaminaLoss(damage * stamina_damage_multiplier) //Take stamina damage equal to half the amount of electrical damage.
+
+	Bolt(user, target,damage,max_bounces,range, user)
+
+/obj/effect/proc_holder/spell/targeted/force_lightning/proc/Bolt(mob/origin,mob/target,bolt_energy,bounces,bolt_range,mob/user = usr)
+
+	if(bolt_energy < 1) //Stop if we would do no damage.
+		return
+	origin.Beam(target,icon_state="lightning[rand(1,12)]",time=5)
+	var/mob/living/current = target
+
+	playsound(get_turf(current), 'sound/magic/lightningshock.ogg', 50, TRUE, -1)
+
+	if(current.anti_magic_check())
+		current.visible_message("<span class='warning'>[current] absorbs the spell, remaining unharmed!</span>", "<span class='userdanger'>You absorb the spell, remaining unharmed!</span>")
+	else
+		current.electrocute_act(bolt_energy,"Force Lightning",1,bolt_flags)
+		spark_system.attach(target)
+		spark_system.start()
+
+	if(bounces < 1)
+		return
+
+	existing_targets += current
+
+	//if(current in existing_targets) //DEBUG
+	//	to_chat(world, "[current] in existing targets.") //DEBUG
+
+	var/list/possible_targets = list()
+	for(var/mob/living/M in view(bolt_range,current))
+		if(user == M) //Don't ask me why I have to arrange the conditional gates in this way; it just does not work if it they're all in the same row with || separators.
+			//to_chat(world, "Target [M] skipped. [M] is caster: [user]") //DEBUG
+			continue
+		else if (!los_check(current,M) )
+			//to_chat(world, "Target [M] skipped; LoS check to [current] failed.") //DEBUG
+			continue
+		else if (origin == M)
+			//to_chat(world, "Target [M] skipped; [M] is origin: [origin].") //DEBUG
+			continue
+		else if (M in existing_targets)
+			//to_chat(world, "Target [M] skipped; [M] is in existing targets.") //DEBUG
+			continue
+		else
+			possible_targets += M
+			//to_chat(world, "Target [M] added.") //DEBUG
+	if(!possible_targets.len)
+		return
+	var/mob/living/next = pick(possible_targets)

```

Comment:
can this rebound back to the caster?

Reply:
No. I will consider changing this; probably when I add the block chance for light saber. I passed on it thus far because it's very niche.

Reply:
No, I mean, can this (unintentionally) choose the CASTER as the second target of the spell/in the chain?

Reply:
No. Impossible.